HW04/Alexey_Zalyotov/spec/my_module_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'my_module'
+
+class Array
+  include MyModule
+end
+
+describe '#my_each' do
+  subject { Array.new([1, 2, 3]) }
+  context 'when no block is given' do
+    it 'returns an Enumerator' do
+      expect(subject.my_each).to be_a(Enumerator)
+      expect(subject.my_each.to_a).to match_array [1, 2, 3]
+    end
+  end
+
+  context 'when reciever is array [1, 2, 3]' do
+    it 'return modified elements in STDOUT' do
+      expect do
+        subject.my_each { |element| puts element * 2 }
```

Comment:
I think it's not necessary for `each` 🤔 what do you think?
```suggestion
        subject.my_each { |element| puts element  }
```

Reply:
Fixed